include/keyence/impl/high_speed_defs.h

```diff
@@ -3,9 +3,28 @@
 
 #include <stdint.h>
 
+#define KEYENCE_FUNDAMENTAL_LENGTH_UNIT (0.01 * 1.0e-6)
+// The integer encoded values representing 'out of range'
+// received from the sensor
+#define KEYENCE_INFINITE_DISTANCE_VALUE -524288
```

Comment:
This is based on Gij's earlier work. The first value you see here is the smallest number that can be represented by the 20 bits that these profiles are packed into. The next one might be indicate a positive versus negative infinity or something like that. I'll need to investigate.